client/layout/sidebar/style.scss

```diff
@@ -1,9 +1,10 @@
 .sidebar {
+	color: var( --sidebar-color );
```

Comment:
This property had to be added in order for the default text to appear in the wanted color schemes color. There are no changes to the default theme as it merely lists the global text color initially derived from the cascade.